### PR TITLE
Glossary additions

### PR DESCRIPTION
**Pull request by bfaloona**

### CODE REVIEW

File: docs/docs/Resources/glossary.md [hunk 1]
```diff
@@ -10,11 +10,21 @@
...
+<b>BG</b> - Blood Glucose
+
+<b>BGI</b> (BG Impact) - The degree to which BG "should" be rising or falling. OpenAPS calculates this value to determine the 'Eventual BG'. This value can be used to make other high/low basal decisions in advanced implementations orf OpenAPS.
+
 <b>Bolus</b> - extra insulin given by a pump, usually to correct for a high BG or for carbohydrates
 
 <b>Basal</b> - baseline insulin level that is pre-programmed into your pump and mimics the insulin your pancreas would give throughout the day and night
 
-<b>IOB</b> - Insulin On Board, or insulin active in your body. Note that most commercially available pumps calculate IOB based on bolus activity only. An OpenAPS implementation calculates and refers most often to net IOB, which takes into account any adjusted (higher or lower) basal rates as well as bolus activity.
+<b>IOB</b> - insulin On Board, or insulin active in your body. Note that most commercially available pumps calculate IOB based on bolus activity only.  Usually, but not always, Net IOB is what Nightscout displays as 'IOB'.
```
Comment (scottleibrand) [inline docs/docs/Resources/glossary.md]: Why lowercase "insulin"?

File: docs/docs/Resources/glossary.md [hunk 2]
```diff
@@ -10,11 +10,21 @@
 
 <b>openaps</b> - the core suite of software tools under development by this community for use in an OpenAPS implementation
 
+<b>BG</b> - Blood Glucose
+
+<b>BGI</b> (BG Impact) - The degree to which BG "should" be rising or falling. OpenAPS calculates this value to determine the 'Eventual BG'. This value can be used to make other high/low basal decisions in advanced implementations orf OpenAPS.
```
Comment (scottleibrand) [inline docs/docs/Resources/glossary.md]: "orf" should be "of"

File: docs/docs/Resources/glossary.md (hunk 1 above)
Comment (bfaloona) [inline docs/docs/Resources/glossary.md]: oops. Fixed.

File: docs/docs/Resources/glossary.md (hunk 2 above)
Comment (bfaloona) [inline docs/docs/Resources/glossary.md]: thanks! Fixed

File: docs/docs/Resources/glossary.md [hunk 3]
```diff
@@ -10,11 +10,21 @@
...
 <b>Basal</b> - baseline insulin level that is pre-programmed into your pump and mimics the insulin your pancreas would give throughout the day and night
 
-<b>IOB</b> - Insulin On Board, or insulin active in your body. Note that most commercially available pumps calculate IOB based on bolus activity only. An OpenAPS implementation calculates and refers most often to net IOB, which takes into account any adjusted (higher or lower) basal rates as well as bolus activity.
+<b>IOB</b> - Insulin On Board, or insulin active in your body. Note that most commercially available pumps calculate IOB based on bolus activity only.  Usually, but not always, Net IOB is what Nightscout displays as 'IOB'.
+
+<b>Net IOB</b> - amount of insulin on board, taking into account any adjusted (higher or lower) basal rates as well as bolus activity. 
+
+<b>Basal IOB</b> - difference (positive or negative) between amount of insulin on board delivered via basal rates, and the amount specified by the profile basal rate.
+
+<b>Treatments IOB</b> - amount of insulin on board delivered via boluses.  Active insulin on
```
Comment (scottleibrand) [inline docs/docs/Resources/glossary.md]: Incomplete?

Comment (bfaloona) [inline docs/docs/Resources/glossary.md]: thanks. Fixed.

**Pull request merged**